### PR TITLE
English layouts

### PR DESCRIPTION
**Pull request by nicoursi**
I created a Terminal layout with symbols. I think a terminal without easy  access to symbols is a bit unconfortable :)

The English qwerty with symbols layout had a few important keys/symbols not easy to reach like: ", ~, and €.

### CODE REVIEW

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
     <word src="becauseyou">because you</word>
     <word src="becuase">because</word>
     <word src="becuse">because</word>
+    <word src="behxviour">behaviour</word>
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: why is that a common typo?

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml [hunk 2]
```diff
@@ -103,6 +104,7 @@
     <word src="isnt">isn't</word>
     <word src="isthe">is the</word>
     <word src="itd">it'd</word>
+    <word src="itslian">italian</word>
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: why is that a common typo?

File: addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml]: I feel like we can just make the changes in the original `terminal` layout. Why do we need two terminal layouts?

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml [hunk 4]
```diff
@@ -23,11 +24,11 @@
         <Key android:codes="d" android:popupCharacters="#%đďḏ" ask:hintLabel="# %"/>
         <Key android:codes="f" android:popupCharacters="^\u0026" ask:hintLabel="^ \u0026"/>
         <Key android:codes="g" android:popupCharacters="`°ĝ" ask:hintLabel="` °"/>
-        <Key android:codes="h" android:popupCharacters="_~ĥḥḫẖ" ask:hintLabel="_ ~"/>
+        <Key android:codes="h" android:popupCharacters="\u005f~" ask:hintLabel="\u005f~"/>
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: what is this change? the popup of `h` is `f`?

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml [hunk 5]
```diff
@@ -4,10 +4,11 @@
           xmlns:ask="http://schemas.android.com/apk/res-auto"
           android:keyWidth="10%p"
           android:keyHeight="@integer/key_normal_height">
+
     <Row>
         <Key android:codes="q" android:popupCharacters="1¹₁" ask:hintLabel="1" android:keyEdgeFlags="left"/>
         <Key android:codes="w" android:popupCharacters="2²₂ŵ" ask:hintLabel="2"/>
-        <Key android:codes="e" android:popupCharacters="3³₃èéêëęē€" ask:hintLabel="3"/>
+        <Key android:codes="e" android:popupCharacters="3€³₃èéêëęē" ask:hintLabel="3"/>
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: for consistency, I'd like to keep this character in the end. We have an on-going discussion on re-ordering.

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 2 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: the "s" and the "a" are next to each other. I make this typo often. But we can remove it.

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 1 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: i made this typo myself and i noted it down. Looking it closely the h and the a are not that close to each other.  I can definately remove it.

File: addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml (hunk 3 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml]: I agree with you. I would do the same with the italian one as well.

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 4 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: No, thats unicode. it's _ and ~. For some reasons "_~ĥḥḫẖ" would not work. You'd never get _ and ~ as first suggestions on long press.

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 5 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: Ok. The reason for that was to have a more useful caracter (€) easier to reach. 
Would it work putting it int the forder xml-it? We europeans use € not $ ;)
I can change it back, no prob

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 4 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: the reason I removed the ĥḥḫẖ, was because I couldn't get  the underscore and the ~ as first choices. No matter what I tried, the only way was to  remove the "ĥḥḫẖ".
Here what happened when you long pressed on h:

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 1 above)
Comment (menny) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: yes, please remove.

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 2 above)
Comment (menny) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: yeah, please remove

File: addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml (hunk 3 above)
Comment (menny) [reply addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml]: thanks

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 5 above)
Comment (menny) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: you can add a `currency_data` file for `it` locale. This is the data we have for Hebrew (`iw` locale): https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/ime/app/src/main/res/values-iw/currency_data.xml

You'll need to create such file under `ime/app/src/main/res/values-it` for Italian locale. Or any other locale you wish.

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 4 above)
Comment (menny) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: I'm confused.. what is the first image and what is the second? I don't want to lose  `ĥḥḫẖ` in the long-press popup. If this is really broken, there is another option to use `android:popupKeyboard`. See https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/english/pack/src/main/res/xml/qwerty.xml#L24 for an example

Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: Thanks! I'll have a look at that.
The first picture is ask master. The second picture is my branch.

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 1 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: done

File: addons/languages/english/pack/src/main/res/xml/english_autotext.xml (hunk 2 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/english_autotext.xml]: done

File: addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml (hunk 3 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/terminal_with_symbols.xml]: Done for terminal. Should I do the same for italian?

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 4 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: Done! I fixed it with your advice

File: addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml (hunk 5 above)
Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/qwerty_with_symbols.xml]: done

File: addons/languages/english/pack/src/main/res/xml/terminal.xml [hunk 6]
```diff
@@ -1,60 +1,47 @@
...
+        <Key android:codes="z" android:popupCharacters="/÷żžź" ask:hintLabel="/"/>
+        <Key android:codes="x" android:popupCharacters="*·×" ask:hintLabel="*"/>
+        <Key android:codes="c" android:popupCharacters="-—çćĉč" ask:hintLabel="−"/>
+        <Key android:codes="v" android:popupCharacters="+±" ask:hintLabel="+"/>
+        <Key android:codes="b" android:popupCharacters="\u003D"/>
+        <Key android:codes="n" android:popupCharacters="&lt;«ńňñ" ask:hintLabel="&lt;"/>
+        <Key android:codes="m" android:popupCharacters="&gt;»µ" ask:hintLabel="&gt;"/>
         <Key android:keyWidth="15%p" android:codes="-5" android:keyEdgeFlags="right" android:isRepeatable="true"/>
     </Row>
-    <!-- the terminal stuff -->
```
Comment (menny) [inline addons/languages/english/pack/src/main/res/xml/terminal.xml]: why did you remove this row? I think that `ESC` and `TAB` keys are used quite a lot in terminal.

Comment (nicoursi) [reply addons/languages/english/pack/src/main/res/xml/terminal.xml]: ooops :) The terminal line is what makes the terminal layout a terminal layout! I don't know how it got removed. Definitely by mistake!

It's back now!
